src/Controller/RequestController.php

```diff
@@ -114,7 +114,7 @@ public function handleBatchRequest(Request $request, array $queries = []) {
     // Walk over all queries and issue a sub-request for each.
     $responses = array_map(function ($query) use ($request) {
       $method = $request->getMethod();
-      $parameters = $method === 'GET' ? $query : [];
+      $parameters = $method === 'GET' ? $query : $request->query->all();
       $content = $method === 'POST' ? json_encode($query) : '';
```

Comment:
@chindris That doesn't seem right. You want to add the query parameters of the main request to $query too if it's a GET request. So basically you want $request->query->all() EXCEPT for "queries" (which would trigger another batch processing). And in case of POST you want the entire POST body/content except of "queries" (same reason).